hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterClientRMService.java

```diff
@@ -19,11 +19,9 @@
 package org.apache.hadoop.yarn.server.router.clientrm;
 
 import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
import java.lang.reflect.InvocationTargetException;:8: Unused import - java.lang.reflect.InvocationTargetException. [UnusedImports]

Reply:
I will fix it.